.github/workflows/swift-toolchain.yml

```diff
@@ -1085,7 +1085,7 @@ jobs:
                 -D CMAKE_Swift_FLAGS="${{ steps.setup-context.outputs.swiftflags }}" `
                 ${{ matrix.cmake_linker_flags }} `
                 -D CMAKE_FIND_PACKAGE_PREFER_CONFIG=YES `
-                -D CMAKE_INSTALL_PREFIX=${{ github.workspace }}/BuildRoot/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain/usr `
+                -D CMAKE_INSTALL_PREFIX=${{ github.workspace }}/BuildRoot/Library/Developer/Toolchains/0.0.0+Asserts/usr `
```

Comment:
Now that I think about it, it might be good to wire this up with `${{ github.inputs.product_version }}` or whatever the input variable is. We currently allow you to specify the toolchain version, but we do not actually write that up here.